google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/CloudStoragePathTest.java

```diff
@@ -231,9 +223,7 @@ public void testToRealPath_hasDotDotDir_throwsIae() throws IOException {
       fs.getPath("a/../b").toRealPath();
       Assert.fail();
     } catch (IllegalArgumentException ex) {
-      assertThat(ex.getMessage())
-          .isEqualTo(
-              "I/O not allowed on dot-dirs or extra slashes when !permitEmptyPathComponents: /a/../b");
+      assertThat(ex.getMessage()).isNotNull();
```

Comment:
the previous assert was fine. I was only commenting on the ones that didn't assert on the message